Functions.Templates/Bindings/userPrompts.json

```diff
@@ -94,12 +94,12 @@
         "id": "blobTrigger-connection",
         "name": "blobTrigger-connection",
         "value": "string",
-        "label": "$blobTrigger_path_label",
-        "help": "$blobTrigger_path_help"
+        "label": "$blobTrigger_connection_label",

```

Comment:
These strings are not present in.
https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Resources/Resources.resx